### PR TITLE
WEB: Moving pandas blog to the website

### PR DESCRIPTION
**Pull request by datapythonista**
Moving the pandas blog posts (now in a separate repo/project) to the `web/community/blog/` in our website. The static site generator we're using for the web, renders them automatically using our layout:

pandas_blog_2

The posts list doesn't have much visual change, just links to the new path:

pandas_blog_1

The main change here, besides adding the posts markdown files (and images) is that the static site generator is now able to not only fetch from rss, but also from the file system.

CC: @TomAugspurger 

Btw @TomAugspurger , I increased the number of posts we show in the list, and see that there are some about dask-ml and other things unrelated to pandas being fetched from your blog. Would be great if you can add a pandas tag to your blog, so we only fetch the relevant content.

### CODE REVIEW

File: web/pandas/community/blog/2019-user-survey.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+Windows users are well represented (see [Steve Dower's talk](https://www.youtube.com/watch?v=uoI57uMdDD4) on this topic).
+
+
+
+
+
+    Linux      61.57%
+    Windows    60.21%
+    MacOS      42.75%
```
Comment (MomIsBestFriend) [inline web/pandas/community/blog/2019-user-survey.md]: I *think* something went wrong, this doesn't adds up to ```100%```

Comment (TomAugspurger) [reply web/pandas/community/blog/2019-user-survey.md]: It was a select multiple, so people could pick more than one.

**Comment by TomAugspurger** [comment]
I'll try to get my tags updated todayl.

**Comment by datapythonista** [comment]
Merging this today if there are no objections.

**Pull request merged**